### ISSUE TITLE
Local view using the container doesn't work on Windows

### ISSUE THREAD

**Issue by esune** [bcgov/devhub-techdocs-publish#11]
When running on Windows and following the [docs to preview content locally](https://github.com/bcgov/devhub-techdocs-publish/blob/main/docs/index.md#how-to-use-the-docker-image-to-preview-content-locally), the command (executed in Git Bash) fails:
```bash
$ docker run -it -p 3000:3000 -v $(pwd):/github/workspace ghcr.io/bcgov/devhub-techdocs-publish preview
Input parameters: preview
Previewer will be started for content after it is generated.
/entrypoint.sh: line 18: cd: /github/workspace: No such file or directory
cp: target '/github/workspace': No such file or directory
No catalog-info file found in repo. Cannot continue.
```
*Note:* the command was run using the following repo, updated with the help of the skeleton builder - https://github.com/bcgov/DITP

The command fails also when running in WSL with the source code checked-out on the Windows filesystem.
Checking-out the code inside the WSL filesystem and running the command yields a success.

**Comment by AErmie** [comment]
We were able to figure out a resolution to this, as follows.

The instructions provided in the [How to use the Docker image to preview content locally](https://github.com/bcgov/devhub-techdocs-publish/blob/main/docs/index.md#how-to-use-the-docker-image-to-preview-content-locally) documentation, were actually written for MAC users (not Windows).

When running the `docker run -it -p 3000:3000 -v $(pwd):/github/workspace ghcr.io/bcgov/devhub-techdocs-publish preview` command on a Windows system (ie. from a Git Bash command prompt), you experience the following error:
 
```shell
$ docker run -it -p 3000:3000 -v $(pwd):/github/workspace ghcr.io/bcgov/devhub-techdocs-publish preview
Input parameters: preview
Previewer will be started for content after it is generated.
/entrypoint.sh: line 18: cd: /github/workspace: No such file or directory
cp: target '/github/workspace': No such file or directory
No catalog-info file found in repo. Cannot continue.
```

When you run docker images you will see 2 different images:

1. spotify/techdocs                        v1.2.4    bf9ca0f99d4e   5 months ago   1.05GB
2. ghcr.io/bcgov/devhub-techdocs-publish   latest    4742cfc58615   7 months ago   2.9GB

The `npx @techdocs/cli serve` command seems to pull the spotify image, whereas the other instructions provided say to use the **ghcr.io/bcgov/devhub-techdocs-publish** image.
 
## Inside the Container

From inside the **devhub-techdocs-publish** image, running the `entrypoint.sh` command directly identified that the mounted directory (ie. $(pwd):/github/workspace) had a permissions error. 
 
Our assumption is that this is due to the external directory being from a Windows system, being mounted into the Linux system.

> NOTE: From additional testing, it seems like it expects the path to be all lowercase, so if your Windows path has any uppercase, this will also cause errors.
 
## On a Windows System

From a Windows system, the following execution works:

1. From a PowerShell command prompt, run `docker run -it -p 3000:3000 ${PWD}:/github/workspace ghcr.io/bcgov/devhub-techdocs-publish preview`
2. From a WSL command prompt, run `docker run -it -p 3000:3000 $(pwd):/github/workspace ghcr.io/bcgov/devhub-techdocs-publish preview`

> [!NOTE]
> This will require the repository to have been cloned within the WSL environment, and not externally in Windows

## Summary

From a Windows system (if the repo is not cloned into WSL), use the PowerShell command prompt (which will also use the latest image specified).

**Comment by AErmie** [comment]
@sheaphillips, the [index.md](https://github.com/bcgov/devhub-techdocs-publish/blob/main/docs/index.md) should be updated to include instructions for Windows users.

As an example, the [README](https://github.com/bcgov/public-cloud-techdocs/blob/dev/README.md#windows-users) file in the `public-cloud-techdocs` repo has been updated accordingly.

**Comment by sheaphillips** [comment]
hi @AErmie . Appreciate your efforts to troubleshoot this for your environment. I can confirm that we had not tested this ourselves in a Windows PowerShell environment. I believe the existing commands will work for Mac, Linux, and WSL as-is. Would you be willing to make a pull request to our README file that includes the modified command for Windows PowerShell?

**Comment by AErmie** [comment]
> hi [@AErmie](https://github.com/AErmie) . Appreciate your efforts to troubleshoot this for your environment. I can confirm that we had not tested this ourselves in a Windows PowerShell environment. I believe the existing commands will work for Mac, Linux, and WSL as-is. Would you be willing to make a pull request to our README file that includes the modified command for Windows PowerShell?

@sheaphillips , I tried making a pull request, but I do not have the permissions to push to this repo.

**Comment by sheaphillips** [comment]
@AErmie that's correct - you wouldn't have the ability to push directly to this repo. You'd create a fork of the repo first, and then initiate pull request from the fork to this repo.

**Comment by AErmie** [comment]
@sheaphillips, PR created: [https://github.com/bcgov/devhub-techdocs-publish/pull/13](https://github.com/bcgov/devhub-techdocs-publish/pull/13)

**Comment by sheaphillips** [comment]
docs now include Windows PowerShell specific instructions.
